### PR TITLE
new method for long press & flag whether or not app needs to be uninstalled 

### PR DESCRIPTION
**Pull request by nirmalashish**

### CODE REVIEW

File: src/main/java/com/znsio/e2e/tools/Driver.java [hunk 1]
```diff
@@ -1,12 +1,13 @@
 package com.znsio.e2e.tools;
 
+import com.google.common.collect.ImmutableMap;
```
Comment (anandbagmar) [inline src/main/java/com/znsio/e2e/tools/Driver.java:3]: are these imports required?

File: src/main/java/com/znsio/e2e/tools/Driver.java [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 import com.znsio.e2e.runner.Runner;
-import io.appium.java_client.AppiumDriver;
-import io.appium.java_client.MobileBy;
-import io.appium.java_client.TouchAction;
+import io.appium.java_client.*;
```
Comment (anandbagmar) [inline src/main/java/com/znsio/e2e/tools/Driver.java:5]: Can you replace with specific imports instead of .*?

Comment (nirmalashish) [reply src/main/java/com/znsio/e2e/tools/Driver.java]: it did it on its own. updated it in the new push

File: src/main/java/com/znsio/e2e/tools/Driver.java (hunk 1 above)
Comment (nirmalashish) [reply src/main/java/com/znsio/e2e/tools/Driver.java:3]: removed

**Review by anandbagmar — APPROVED** [review]

**Pull request merged**